transfer/streaming/streaming-01-http-to-http/README.md

```diff
@@ -102,11 +111,11 @@ curl -H 'Content-Type: application/json' -d @transfer/streaming/streaming-01-htt
 ```
 
 ### Start the transfer
-First we need to set up the receiver server on the consumer side that will receive a call for every message. For this
+First we need to set up the logging webserver on the consumer side that will receive a call for every message. For this
 you'll need to open another terminal shell and run:
 ```shell
-./gradlew util:http-request-logger:build
-HTTP_SERVER_PORT=4000 java -jar util/http-request-logger/build/libs/http-request-logger.jar
+docker build -t http-request-logger util/http-request-logger
+docker run -p 4000:4000 http-request-logger
```

Comment:
Not sure why running with docker should be better than running the jar directly

Reply:
First, I could not run this JAR after building it. Unfortunately, I could not fix this error:
```
Error: Could not find or load main class org.eclipse.edc.samples.util.HttpRequestLoggerServer
Caused by: java.lang.ClassNotFoundException: org.eclipse.edc.samples.util.HttpRequestLoggerServer
```

Secondly, there is a Dockerfile that is already used in other samples

Reply:
looks like the jar task in the build.gradle.kts misses to include `.class` files in the jar:
```java
from(sourceSets["main"].output)
```

Reply:
Thanks for your help! Now running the JAR works. I have changed the use of container back to JAR.